packages/lib/fixtures/test_pdf

```diff
@@ -0,0 +1,198 @@
+%PDF-1.3

```

Comment:
Support test files should go in `packages/app-cli/tests/support` which you can then access using the `supportDir` constant